packages/server/README.md

```diff
@@ -76,3 +76,7 @@ Prepend `SNAPSHOT_UPDATE=1` to any test command. See [`snap-shot-it` instruction
 SNAPSHOT_UPDATE=1 yarn test test/unit/api_spec.js
 SNAPSHOT_UPDATE=1 yarn test test/integration/cli_spec.js
 ```
+
+### V8 Snapshots
```

Comment:
Should you give a little explainer here about what v8 snapshots are?

Reply:
Added here: [cypress-io/cypress@`dc98498` (#23732)](https://github.com/cypress-io/cypress/pull/23732/commits/dc98498362a72d91d35c3a0d2f31ba201734ebc7)